op-service/client/client.go

```diff
@@ -246,7 +246,7 @@ func (ic *InstrumentedClient) SuggestGasPrice(ctx context.Context) (*big.Int, er
 }
 
 func (ic *InstrumentedClient) SuggestGasTipCap(ctx context.Context) (*big.Int, error) {
-	return instrument2[*big.Int](ic.m, "eth_maxPriorityFeePerGas", func() (*big.Int, error) {
+	return instrument2[*big.Int](ic.m, "eth_gasPrice", func() (*big.Int, error) {
```

Comment:
hey @franciscotobar, I don't think this is a good replacement.

`eth_maxPriorityFeePerGas` seems to be a tip, on top of the regular `eth_gasPrice`, in order to get a transaction included. If we set it to `eth_gasPrice` we are "doubling" the used price. Probably we can apply a percentage here, like 10% (if I am not mistaken it is a common practice to bump price by at least 10%).

Or am I misunderstanding something?

Reply:
I handled this in my PR (on top of this one) so we are good.